### DISCUSSION TITLE
Cisco Monitor Port

### DISCUSSION THREAD

**Discussion by tony-advent** [netbox-community/netbox#12998] [Help Wanted!]
Hi,

I would like to record the configuration of a Monitor of a cisco switch.

I have a Armis device Monitoring the ports on a Cisco Switch with the following configuration:

#################

interface GigabitEthernet1/0/23
 description Armis network port
 switchport mode access
!
interface GigabitEthernet1/0/24
 description Armis SPAN port
!

monitor session 2 source interface Gi1/0/1 - 23 , Gi1/0/25 - 48
monitor session 2 destination interface Gi1/0/24

end

=====================================================
Monitor session 2
---------
Type                   : Local Session
Source Ports           :
    Both               : Gi1/0/1-23,Gi1/0/25-48
Destination Ports      : Gi1/0/24
    Encapsulation      : Native
          Ingress      : Disabled

######################

How do I recorded the Monitor Config within NetBox?

**Comment by abhi1693** [comment]
You can either create a plugin or custom fields. There aren't any specific fields to add this information in the core code.

**Comment by tony-advent** [comment]
Thanks
